folding/src/instance_witness.rs

```diff
@@ -82,6 +86,19 @@ impl<G: CommitmentCurve, I: Instance<G>> ExtendedInstance<G, I> {
             extended: vec![],
         }
     }
+    /// wraps the inner absorb, adding its own commitments
+    fn to_absorb(&self) -> (Vec<G::ScalarField>, Vec<G::BaseField>) {
+        let mut elements = self.inner.to_absorb();
+        let extended_commitments = self.extended.iter().flat_map(|commit| {
+            assert_eq!(commit.elems.len(), 1);
+            let point = commit.elems[0];
+            //this may need change if we need to support the infinity point
```

Comment:
Why this case? Can we not simply use absorb_commitment, please?

Reply:
we could, but I was not sure about its implementation, and is one extra thing to duplicate in IVC, but I have no issue with changing it if you think is better.

Reply:
another option is to change the signature to allow only one field, and leave it up to the trait implementation to decompose and represent the points in the native field.